AppleStore/AppleStore/BasketViewController/FavoritesViewController.swift

```diff
@@ -7,6 +7,212 @@
+        sentLabel.font = UIFont.systemFont(ofSize: 12)
+        orderView.addSubview(sentLabel)
+
+        deliveredLabel.frame = CGRect(x: 290, y: 140, width: 100, height: 25)
+        deliveredLabel.text = "Доставлено"
+        deliveredLabel.textColor = .gray
+        deliveredLabel.font = UIFont.systemFont(ofSize: 12)
+        orderView.addSubview(deliveredLabel)
+    }
+
+    private func setupNotificationSection() {
+        recommendedLabel.frame = CGRect(x: 20, y: 450, width: 300, height: 25)
+        recommendedLabel.text = "Рекомендуется вам"
+        recommendedLabel.font = UIFont.boldSystemFont(ofSize: 22)
+        view.addSubview(recommendedLabel)
+
+        notificationImageView.frame = CGRect(x: 40, y: 500, width: 50, height: 50)
+        notificationImageView.image = notificationImage
+        view.addSubview(notificationImageView)
+
+        notificationLabel.frame = CGRect(x: 110, y: 495, width: 250, height: 75)
+        notificationLabel.text = "Получайте новости о своем заказе в режиме реального времени."
+        notificationLabel.numberOfLines = 3
+        notificationLabel.font = UIFont.boldSystemFont(ofSize: 16)
+        view.addSubview(notificationLabel)
+
+        turnOnNotificationLabel.frame = CGRect(x: 110, y: 560, width: 300, height: 50)
+        turnOnNotificationLabel.text = "Включите уведомления, чтобы получать новости о своем заказе."
+        turnOnNotificationLabel.numberOfLines = 2
+        turnOnNotificationLabel.textColor = .gray
+        turnOnNotificationLabel.font = UIFont.systemFont(ofSize: 13)
+        view.addSubview(turnOnNotificationLabel)
+
+        forwardOnNotificationButton.frame = CGRect(x: 380, y: 530, width: 15, height: 25)
+        forwardOnNotificationButton.setBackgroundImage(UIImage(systemName: "chevron.forward"), for: .normal)
+        forwardOnNotificationButton.tintColor = .gray
+        view.addSubview(forwardOnNotificationButton)
+    }
+
+    private func setupYourDeviceSection() {
+        demarcationLineBetweenUnderLabelsView.frame = CGRect(x: 20, y: 650, width: 380, height: 1)
+        demarcationLineBetweenUnderLabelsView.backgroundColor = .systemGray3
+        view.addSubview(demarcationLineBetweenUnderLabelsView)
+
+        yourDeviceLabel.frame = CGRect(x: 20, y: 700, width: 300, height: 40)
+        yourDeviceLabel.text = "Ваши устройства"
+        yourDeviceLabel.font = UIFont.boldSystemFont(ofSize: 25)
+        view.addSubview(yourDeviceLabel)
+
+        showAllLabel.frame = CGRect(x: 270, y: 710, width: 150, height: 25)
+        showAllLabel.text = "Показать все"
+        showAllLabel.textColor = .systemBlue
+        view.addSubview(showAllLabel)
+    }
 }
+
+extension FavoritesViewController: UIImagePickerControllerDelegate {
+    func imagePickerController(_ picker: UIImagePickerController,
+                               didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        picker.dismiss(animated: true)
```

Comment:
```suggestion
        dismiss(animated: true)
```
и его вызываем всегда в конце уже